packages/utilities/src/lib/pickRandom.ts

```diff
@@ -0,0 +1,12 @@
+/**
+ * Picks a random element from an array
+ * @param array The array to pick a random element from
+ */
+export function pickRandom<T>(array: readonly T[], count?: 1): T;
+export function pickRandom<T>(array: readonly T[], count: number): T[];
+export function pickRandom<T>(array: readonly T[], count = 1): T | T[] {
+	const arr = [...array];
+	if (typeof count === 'undefined') return arr[Math.floor(Math.random() * arr.length)];
```

Comment:
```suggestion
	if (typeof count === 'undefined' || count === 1) return arr[Math.floor(Math.random() * arr.length)];
```
or change the typings

Reply:
I'm pushing a typing change